package.json

```diff
@@ -25,5 +25,14 @@
     "eslint": "^8.2.0",
     "eslint-plugin-node-core": "iansu/eslint-plugin-node-core",
     "tape": "^5.2.2"
+  },
+  "files": [
+    "index.js",
+    "primordials.js"
+  ],
```

Comment:
i strongly prefer not to use the "files" field - it's very dangerous. let's stick with `.npmignore` here.